src/test/crimson/perf_crimson_msgr.cc

```diff
@@ -74,8 +80,16 @@ static seastar::future<> run(unsigned rounds,
                                     MessageRef m) override {
         ceph_assert(m->get_type() == CEPH_MSG_OSD_OP);
         // reply
-        Ref<MOSDOp> req = boost::static_pointer_cast<MOSDOp>(m);
-        return c->send(MessageRef{ new MOSDOpReply(req.get(), 0, 0, 0, false), false });
+        const static pg_t pgid;
+        const static object_locator_t oloc;
+        const static hobject_t hobj(object_t(), oloc.key, CEPH_NOSNAP, pgid.ps(),
+                                    pgid.pool(), oloc.nspace);
+        static spg_t spgid(pgid);
+        MOSDOp *rep = new MOSDOp(0, 0, hobj, spgid, 0, 0, 0);
```

Comment:
could use a comment explaining why we want to use `MOSDOp` instead of `MOSDReply` for testing.

Reply:
added `// server replies with MOSDOp to generate server-side write workload`